docs/TYPESCRIPT.md

```diff
@@ -64,8 +65,3 @@ push.on('notification', (data: my.custom.NotificationEventResponse) => {
 	console.log(data.additionalData.bacon);
 });
 ```
-
-## Outdated definitions
-
-Is our definition file at DefinitelyTyped outdated? Is there any improvements that could be done?
-We welcome any contribution, and they should be done through issues created [there](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/new).
```

Comment:
I'd say we're better off just switching `and they should be done through issues created *here*` instead of removing this part, even though I agree that it's kinda implied for the whole project/repo. What do you say?

Reply:
Was thinking the same first, but now that the declarations are part of the source... so I don't think the definitions should ever be considered outdated again. If there is an issue with the definitions, it's just a bug now, I guess.

Reply:
I can agree with that. I'll make room on schedule to bring the definitions up to par with the actual API/source ASAP so that we're good from then on.